### PR TITLE
Add new eslint config

### PR DESCRIPTION
**Pull request by Alex4Max**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,14 +1,34 @@
 {
-  "extends": "airbnb",
+  "parser": "babel-eslint",
   "env": {
     "browser": true,
     "node": true,
-    "serviceworker": true
```
Comment (sleepwalky) [inline .eslintrc]: Why have you deleted this line?

File: .eslintrc [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+    "mocha": true
   },
+  "extends": "airbnb",
   "rules": {
-    "linebreak-style": 0,
-    "react/jsx-filename-extension": 0,
-    "react/prop-types": 1,
-    "no-multiple-empty-lines": ["error", { "max": 1, "maxEOF": 0 }]
```
Comment (sleepwalky) [inline .eslintrc:12]: and this line

File: .eslintrc [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
+    "consistent-return": "off",
+    "new-cap": ["error", { "capIsNewExceptions": ["Router", "Schema", "URLSlugs"] }],
+    "no-underscore-dangle": ["error", { "allow": ["_id"] }],
+    "arrow-parens": ["error", "as-needed"],
+    "linebreak-style": "off",
+    "no-mixed-operators": "off",
+    "class-methods-use-this": "off",
+    "react/forbid-prop-types": "off",
+    "react/jsx-filename-extension": "off",
+    "react/jsx-indent": "off",
```
Comment (sleepwalky) [inline .eslintrc]: Why?

File: .eslintrc [hunk 4]
```diff
@@ -1,14 +1,34 @@
...
-    "serviceworker": true
+    "mocha": true
   },
+  "extends": "airbnb",
   "rules": {
-    "linebreak-style": 0,
-    "react/jsx-filename-extension": 0,
-    "react/prop-types": 1,
-    "no-multiple-empty-lines": ["error", { "max": 1, "maxEOF": 0 }]
+    "comma-dangle": ["error", "never"],
```
Comment (sleepwalky) [inline .eslintrc]: Why?

File: .eslintrc [hunk 5]
```diff
@@ -1,14 +1,34 @@
...
+    "no-underscore-dangle": ["error", { "allow": ["_id"] }],
+    "arrow-parens": ["error", "as-needed"],
+    "linebreak-style": "off",
+    "no-mixed-operators": "off",
+    "class-methods-use-this": "off",
+    "react/forbid-prop-types": "off",
+    "react/jsx-filename-extension": "off",
+    "react/jsx-indent": "off",
+    "react/self-closing-comp": "off",
+    "react/no-danger": "off",
```
Comment (sleepwalky) [inline .eslintrc]: Why?

File: .eslintrc [hunk 6]
```diff
@@ -1,14 +1,34 @@
...
+    "react/no-string-refs": "off",
```
Comment (sleepwalky) [inline .eslintrc]: Should be turned on

File: .eslintrc [hunk 7]
```diff
@@ -1,14 +1,34 @@
...
+    "react/jsx-no-target-blank": "off",
+    "react/jsx-no-duplicate-props": "off",
+    "jsx-a11y/label-has-for": "off",
+    "jsx-a11y/no-static-element-interactions": "off",
+    "import/no-named-as-default": "off",
+    "import/extensions": "off",
+    "import/imports-first": "off",
```
Comment (sleepwalky) [inline .eslintrc]: Why don't you like this one?

File: .eslintrc (hunk 2 above)
Comment (Alex4Max) [reply .eslintrc:12]: now it locates below

**Review by sleepwalky — APPROVED** [review]

**Pull request merged**